tests/Behat/FeatureContext.php

```diff
@@ -298,4 +299,20 @@ public function assertVisibility($element): void {
     $this->assertVisuallyVisible($node);
   }
 
+  /**
+   * Create a dummy stakeholder.
+   *
+   * @todo Make it more flexible.
```

Comment:
drop this comment, and add example usage

Reply:
Previously fixed